### PR TITLE
Update recommended-settings-for-eop-and-office365-atp.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a default tab for Anti phishing policies

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3452

File: microsoft-365/security/office-365-security/recommended-settings-for-eop-and-office365-atp.md [hunk 1]
```diff
@@ -184,11 +184,11 @@ Note that these are the same settings that are available in [anti-spam policy se
 
 ****
 
-|Security feature name|Standard|Strict|Comment|
-|---|---|---|---|
-|**Enable anti-spoofing protection** <p> _EnableAntispoofEnforcement_|On <p> `$true`|On <p> `$true`||
-|**Enable Unauthenticated Sender** <p> _EnableUnauthenticatedSender_|On <p> `$true`|On <p> `$true`|Adds a question mark (?) to the sender's photo in Outlook for unidentified spoofed senders. For more information, see [Spoof settings in anti-phishing policies](set-up-anti-phishing-policies.md).|
-|**If email is sent by someone who's not allowed to spoof your domain** <p> _AuthenticationFailAction_|**Move message to the recipients' Junk Email folders** <p> `MoveToJmf`|**Quarantine the message** <p> `Quarantine`|This setting applies to blocked senders in [spoof intelligence](learn-about-spoof-intelligence.md).|
+|Security feature name|SDefault|Standard|Strict|Comment|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/recommended-settings-for-eop-and-office365-atp.md]:
```suggestion
|Security feature name|Default|Standard|Strict|Comment|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**